pages/code_schools.js

```diff
@@ -104,14 +104,13 @@ export default class CodeSchools extends React.Component {
     const vaApproved = allSchools.filter(school =>
       school.locations.some(location => location.vaAccepted),
     );
-
     this.setState({ filteredSchools: vaApproved, selectedStates: [] });
   };
 
   filterVetTecApproved = () => {
     const { allSchools } = this.props;
-    const vetTecApprovedSchools = allSchools.filter(school => school.isVetTecApproved);
-    this.setState({ filteredSchools: vetTecApprovedSchools, selectedStates: [] });
+    const vetTecApproved = allSchools.filter(school => school.isVetTecApproved);
+    this.setState({ filteredSchools: vetTecApproved, selectedStates: [] });
```

Comment:
I'd prefer the old name because the data structure is embedded as part of it (schools - an array of schools), but you don't need to revert this change. Just explaining why it was done that way!

Reply:
I see why you chose the old name, and I do think the old name is more fitting. When I looked at it originally I was trying to make it resemble the vaApproved filter as much as I could since they both represent filters for a specific program- but now I know the data for vaApproved is dependent on location, the data for vetTec is dependent on the school. Makes sense! Thanks. Reverted back to the OG!